chalicelib_smaht/check_schedules.py

```diff
@@ -65,6 +67,16 @@ def morning_checks(event):
     app.core.queue_scheduled_checks('all', 'morning_checks')
 
 
+@schedule(SCHEDULES, stage=STAGE, disabled_stages=DISABLED_STAGES)
+def morning_checks_2(event):
+    app.core.queue_scheduled_checks('all', 'morning_checks_2')
```

Comment:
Is there anything on morning checks two right now? Is this omitted below? Don't want to double queue